webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -13,12 +13,14 @@ export const get = async (payload, url, store, actionTypes) => {
     );
     store.dispatch({
       type: actionTypes.SUCCESS,
-      payload: { ...payload, ...data },
+      payload,
+      response: data,
     });
   } catch (error) {
     store.dispatch({
       type: actionTypes.FAILURE,
-      payload: { ...payload, error },
+      payload,
+      response: { error },
```

Comment:
Would it make sense changing it to `response: error`?
For the same reason you don't do `response: { data }`

So if the type is `SUCCESS` you will receive the data as the `response` and if the type is `FAILURE` you will receive the error as the `response`.

Reply:
done